main.py

```diff
@@ -1,108 +1,128 @@
 import os
-from dotenv import load_dotenv
-from discord.ext import commands
+import asyncio
```

Comment:
Alphabetical ordering is most align with PEP8 standards so would suggest placing asyncio above os import.